src/main/java/com/paneedah/mwc/MWC.java

```diff
@@ -135,6 +142,11 @@ public void init(FMLInitializationEvent initializationEvent) {
         CHANNEL.registerMessage(new WorkbenchServerMessageHandler(), WorkbenchServerMessage.class, 12, Side.SERVER);
         CHANNEL.registerMessage(new CraftingServerMessageHandler(), CraftingServerMessage.class, 13, Side.SERVER);
         CHANNEL.registerMessage(new EntityPickupMessageHandler(), EntityPickupMessage.class, 14, Side.SERVER);
+
+        /*
+        *   Load Recipes tad bit earlier so GS can add its own recipes.
+         */
```

Comment:
This should be a comment not a Javadoc

Reply:
done in [1f43aa2](https://github.com/Cubed-Development/Modern-Warfare-Cubed/pull/303/commits/1f43aa26210a81a0627894a9a0dbaa87e0a1da56)